frontend/packages/integration-tests-cypress/tests/crud/annotations.spec.ts

```diff
@@ -0,0 +1,193 @@
+import { checkErrors, testName } from '../../support';
+import { nav } from '../../views/nav';
+import { listPage } from '../../views/list-page';
+import * as yamlEditor from '../../views/yaml-editor';
+import { errorMessage } from '../../views/form';
+import { detailsPage } from '../../views/details-page';
+import { modal } from '../../views/modal';
+
+const createExampleConfigMapInstance = () => {
+  nav.sidenav.clickNavLink(['Workloads', 'Config Maps']);
+  cy.byLegacyTestID('resource-title').should('have.text', 'Config Maps');
+  listPage.clickCreateYAMLbutton();
+  cy.byTestID('resource-sidebar').should('exist');
+  yamlEditor.isLoaded();
+  yamlEditor.clickSaveCreateButton();
+  cy.get(errorMessage).should('not.exist');
+};
+
+const deleteExampleConfigMapInstance = () => {
+  detailsPage.isLoaded();
+  detailsPage.titleShouldContain('example');
+  detailsPage.clickPageActionFromDropdown('Delete Config Map');
+  modal.shouldBeOpened();
+  modal.submit();
+  modal.shouldBeClosed();
+  cy.get(errorMessage).should('not.exist');
+};
+
+const getNameValueEditorRow = (row: number) => {
+  return cy.byTestID('pairs-list-row').eq(row - 1);
+};
+
+const setName = (row: JQuery<HTMLElement>, nameValue: string) => {
+  cy.wrap(row).within(() => {
+    cy.byTestID('pairs-list-name').type(nameValue);
+  });
+};
+const setValue = (row: JQuery<HTMLElement>, value: string) => {
+  cy.wrap(row).within(() => {
+    cy.byTestID('pairs-list-value').type(value);
+  });
+};
+/* const clearName = (row: JQuery<HTMLElement>) => {
```

Comment:
We should remove commented out code if not needed.

Reply:
fixed